src/Microsoft.Diagnostics.Runtime/src/Common/ModuleInfo.cs

```diff
@@ -11,12 +11,14 @@ namespace Microsoft.Diagnostics.Runtime
     /// <summary>
     /// Provides information about loaded modules in a DataTarget.
     /// </summary>
-    public class ModuleInfo
+    public class ModuleInfo : IDisposable
```

Comment:
If possible, I'd like to arrive at a design that doesn't require ModuleInfo to be disposable.  It would be acceptable if we stopped caching the instance here and returned a new PEImage every call to OpenPEImage (or whatever I called that function).  In fact, that might solve a lot of other problems as well internally with having to worry about PEImage thread safety.

I would suggest not tackling that problem in this PR.  Instead leave ModuleInfo as non-IDisposable and file an issue for me to clean this up when the PR is complete.

Reply:
Perfect, filed this as https://github.com/microsoft/clrmd/issues/465.